.travis.yml

```diff
@@ -29,6 +29,7 @@ before_install:
 # Download and unpack the stack executable
 - mkdir -p ~/.local/bin
 - export PATH=$HOME/.local/bin:$PATH
+- export PATH=$PATH:/usr/lib/postgresql/10/bin/
```

Comment:
We need this in order for the `tmp-postgres` to find `initdb`. Ubuntu doesn't have `initdb` on the path by default.